### DISCUSSION TITLE
Wondering if mod could be used as x-ray

### DISCUSSION THREAD

**Discussion by giorov** [maruohon/minihud#249] [Q&A] [answered]
Just curious, would it be possible to x-ray ores with this? I want to draw spheres for spawn proofing farms on a server that doesn't allow xraying.

**Comment by maruohon** [comment, marked as answer]
No, MiniHUD shouldn't have anything that would allow "X-Raying". Unless you count the structure bounding boxes in far-away chunks in single player (when the chunk isn't loaded on the client yet), or in the "render through" mode which would allow you to see where some structures are underground.

**Comment by giorov** [comment]
Thanks, and those are just in single player then, you'd need access to the seed in multiplayer for stuff like slime chunks I bet.

**Reply by maruohon** [reply]
Yes you would need to input the seed for MiniHUD to show the slime chunks, and for structure bounding boxes the server would need to have a mod to send that data to MiniHUD.

**Reply by giorov** [reply]
Perfect, thanks!
